src/ui/components/results/directanswercomponent.js

```diff
@@ -86,8 +86,10 @@ export default class DirectAnswerComponent extends Component {
    * we want to wire up the behavior for interacting with the quality feedback reporting (thumbsup/down)
    */
   onMount () {
-    // Avoid bindings if the feedback has previously been submitted
-    if (this.getState('feedbackSubmitted') === true) {
+    const isUsingCustomCard = this._config.defaultCard;
+    const feedbackSubmitted = this.getState('feedbackSubmitted') === true;
+    // Avoid bindings if the feedback has previously been submitted or is using a custom card.
```

Comment:
Have you and @alexisgrow talked about where these bindings should live for a custom card (here vs the Theme)? We are going to want most, if not all, of these bindings for a custom direct answers card as well. These bindings are handle RTF click analytics, form submission, and submission of feedback. I think any default direct answers card in the Theme will need these bindings. We could have this component handle the bindings. We'd just check for the existence of the related DOM element before doing the binding.

Reply:
hmm, I thought it might be clearer to do this in the theme, but I guess we pass down all of the selectors through the config

Reply:
We can discuss as a group if you'd like! I just think it's important that all parties are on the same page about where this will live.

Reply:
okay, I think we agreed on letting the base theme component do the bindings!